tests/python/gpu/test_forward.py

```diff
@@ -1,61 +0,0 @@
-import os
```

Comment:
Its here to gurrentee numerical consistency between cpu and gpu implementation. Error may not be big enough to change acc.